sdk/postgresql/arm-postgresql-flexible/src/models/mappers.ts

```diff
@@ -624,8 +624,8 @@ export const Storage: coreClient.CompositeMapper = {
           name: "String"
         }
       },
-      iopsTier: {
-        serializedName: "iopsTier",
+      tier: {
```

Comment:
I don't see this breaking in the changelog, why?

Reply:
because last stable doesn't have this